Jenkinsfile

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env groovy
+
+REPOSITORY = 'maslow'
+
+node {
+  def govuk = load '/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy'
+
+  try {
+    stage("Checkout") {
+      checkout scm
+    }
+
+    stage('Clean') {
+      govuk.cleanupGit()
+    }
+
+    stage('Env') {
+      govuk.setEnvar('RAILS_ENV', 'test')
+    }
+
+    stage('Bundle') {
+      govuk.bundleApp()
+    }
+
+    stage('Lint') {
+      govuk.rubyLinter()
+    }
+
+    stage("Build") {
+      govuk.runRakeTask('db:mongoid:drop')
+      sh "bundle exec rake"
+    }
+
+    stage("Result") {
+      govuk.pushTag(REPOSITORY, env.BRANCH_NAME, 'release_' + env.BUILD_NUMBER)
+      govuk.deployIntegration(REPOSITORY, env.BRANCH_NAME, 'release', 'deploy')
+    }
```

Comment:
I've been suggesting to people to make the stage conditional (e.g. [1]) if all the operations in it are conditional on the branch, as this is far less surprising both in the Jenkinsfile, and in the Jenkins web ui. As result is such a generic name, that's not as applicable here, but I thought I'd bring the option up.

1: https://github.com/alphagov/gds-api-adapters/blob/master/Jenkinsfile#L121